CLI - `tk update test` isn't working at all

**Describe the bug**
It's not possible to use `tk update test` at all.

```
$ tk update test      
panic: unable to redefine 'c' shorthand in "test" flagset: it's already used for "command" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc000053300, 0xc000a82fa0)
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:874 +0x3ec
github.com/spf13/pflag.(*FlagSet).AddFlagSet.func1(0xc000a82fa0)
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:887 +0x65
github.com/spf13/pflag.(*FlagSet).VisitAll(0x4?, 0xc00125fa98)
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:290 +0xe3
github.com/spf13/pflag.(*FlagSet).AddFlagSet(0xc000a8ef00?, 0x0?)
        /home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:885 +0x3d
github.com/spf13/cobra.(*Command).mergePersistentFlags(0xc000a8ef00)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:1674 +0x65
github.com/spf13/cobra.(*Command).InitDefaultHelpFlag(0x1?)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:1048 +0x25
github.com/spf13/cobra.(*Command).execute(0xc000a8ef00, {0x3037f28, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:792 +0x105
github.com/spf13/cobra.(*Command).ExecuteC(0x2fd4100)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/kubeshop/testkube/cmd/kubectl-testkube/commands.Execute()
        /home/runner/work/testkube/testkube/cmd/kubectl-testkube/commands/root.go:122 +0x3dd
main.main()
        /home/runner/work/testkube/testkube/cmd/kubectl-testkube/main.go:24 +0x17
```
The same for `tk update test --name curl-test`,  `tk update test --name curl-test`, etc.

But, everything looks fine for testsource, or testsuite
 ```
$ tk update testsource
⨯ pass valid name (in '--name' flag)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run 'tk update test'
2. Check the output

**Version / Cluster**
Client Version 1.5.39
Server Version v1.5.39

--- a/cmd/kubectl-testkube/commands/tests/update.go
+++ b/cmd/kubectl-testkube/commands/tests/update.go
@@ -102,7 +102,7 @@ func NewUpdateTestsCmd() *cobra.Command {
 	cmd.Flags().StringArrayVarP(&copyFiles, "copy-files", "", []string{}, "file path mappings from host to pod of form source:destination")
 	cmd.Flags().StringVarP(&image, "image", "i", "", "if uri is git repository we can set additional branch parameter")
 	cmd.Flags().StringArrayVar(&imagePullSecretNames, "image-pull-secrets", []string{}, "secret name used to pull the image in container executor")
-	cmd.Flags().StringArrayVarP(&command, "command", "c", []string{}, "command passed to image in container executor")
+	cmd.Flags().StringArrayVarP(&command, "command", "", []string{}, "command passed to image in container executor")
 
 	return cmd
 }

